x-pack/plugins/canvas/canvas_plugin_src/functions/common/columns.ts

```diff
@@ -14,6 +14,28 @@ interface Arguments {
   exclude: string;
 }
 
+const prepareFields = (fields: string) => fields.split(',').map((field) => field.trim());
+
+const getFieldsIds = (cols: DatatableColumn[]) => cols.map((col) => col.id ?? col.name);
+
+const splitColumnsByFields = (
+  cols: DatatableColumn[],
+  fields: string[],
+  saveOther: boolean = false
+) =>
+  cols.reduce<{ matched: DatatableColumn[]; other: DatatableColumn[] }>(
+    (splitedColumns, col) => {
```

Comment:
NIT: `splitColumns` is a more proper name

Reply:
Done.